Distribution/Server/Framework/Auth.hs

```diff
@@ -155,6 +158,20 @@ checkPriviledged users uid (IsUserId uid':others) =
 
 checkPriviledged _ _ (AnyKnownUser:_) = return True
 
+------------------------------------------------------------------------
+-- Key auth method
+--
+
+-- | Handle a key auth request
+checkKeyAuth :: Users.Users -> BS.ByteString -> Either AuthError (UserId, UserInfo)
```

Comment:
Can we give it a clearer name? e.g. something using words like "access token" or "API token" or whatever this is commonly known as so it's clear to readers?